### DISCUSSION TITLE
contribution to phi from zwt obs in glm

### DISCUSSION THREAD

**Discussion by wkitlasten** [usgs/pestpp#243] [Ideas]
Not a big deal, but since I have been convinced to carry many predictions in my pest builds, would it be easy enough to turn off the "contribution to phi from" for zwt obs in the screen output for glm runs? In most of my cases it blows all other reporting off the screen, plus makes a "quick check" in the rec file a bit painful.

**Comment by jtwhite79** [comment]
Yes this should be cleaned up
